server/server/migrations/migrations/add_divisions_010.py

```diff
@@ -1,20 +0,0 @@
-from common.arangodb import get_db
-from migrations.base import Migration
-
-
-class SecondMigration(Migration):
-    migration_id = 'add_divisions_010'
-    tasks = ['create_divisions_collection', 'create_text_divisions_collection']
-
-    def create_divisions_collection(self):
-        db = get_db()
-
-        paragraphs = db.create_collection('divisions')
-        paragraphs.add_hash_index(fields=['uid'], unique=True)
-
-    def create_text_divisions_collection(self):
-        db = get_db()
-
-        paragraphs = db.create_collection('text_divisions')
-        paragraphs.add_hash_index(fields=['uid'], unique=True)
-        paragraphs.add_hash_index(fields=['division'], unique=False)
```

Comment:
In general, I am not sure that we should delete the migration file, as far as I know, it is not quite right :thinking:

Reply:
There doesn't seem to be any code that references the two collections at the moment, but we can keep it and delete it later when we're sure it's no longer needed.

Reply:
Thanks for bringing him back, he didn't deserve this fate =D

it's not that we need it now, it's just bad practice, maybe later I can say more
personally, I think we need to know what was created when, in what form, for what and who created it + may need additional migration in the future to remove collections that were created here